### PR TITLE
Example

### PR DESCRIPTION
**Pull request by markelog**
As discussed in https://github.com/jscs-dev/node-jscs/issues/1106

Requires https://github.com/jscs-dev/node-jscs/pull/1717 changes, which should be soon released with the 2.1.1

@zxqfox can you create a docs basing on https://github.com/jscs-dev/node-jscs/pull/1717 tests and this example?

@sindresorhus would be cool if you would check out those docs after they done

/cc @hzoo

### CODE REVIEW

File: .jscs.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+	"preset": "./jscsrc-esnext",
```
Comment (markelog) [inline .jscs.json]: just as an example, you could also include rules, filters, etc; locally or from your dependencies

File: package.json [hunk 2]
```diff
@@ -50,15 +51,14 @@
...
   "peerDependencies": {
-    "jscs": "^2.1.0"
+    "jscs": "^2.1.1"
   },
   "xo": {
     "ignores": [
       "fixture.js"
     ]
-  }
+  },
```
Comment (noamokman) [inline package.json]: Not that is matters much, but this line drops the CI

File: .jscs.json [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
 	"disallowCapitalizedComments": true,
 	"disallowEmptyBlocks": true,
 	"disallowIdentifierNames": [
```
Comment (sindresorhus) [inline .jscs.json:5]: Why the name change of this file?

**Comment by sindresorhus** [comment]
How could I export multiple presets?

In ESLint I would have a main export, but also be able to use the files in root.

The user could do:

```
{
  "extends": "xo"
}
```

Which would get the config from the `main` field in package.json.

But also:

```
{
  "extends": "xo/esnext"
}
```

Which get's the `esnext.{js,json}` file from root.

File: .jscs.json (hunk 3 above)
Comment (markelog) [inline .jscs.json:5]: Corrected the `main` field in the `package.json`

**Comment by markelog** [comment]
> How could I export multiple presets?

Good catch, https://github.com/jscs-dev/node-jscs/pull/1717 updated

**Comment by sindresorhus** [comment]
@markelog Can you update this PR to export multiple presets? Not quite clear from the referenced PR how I would do it.

**Comment by markelog** [comment]
The same way you shown, pr updated though

**Comment by markelog** [comment]
To be clear, consumer of your preset would write - 

``` js
{
  "preset": "xo/jscsrc-esnext"
}
```

**Comment by markelog** [comment]
@sindresorhus we about to release a new version, is everything looks good?

**Comment by hzoo** [comment]
You should update the readme too

**Comment by markelog** [comment]
If @sindresorhus is still interested in this project, i will

**Comment by sindresorhus** [comment]
@markelog No need. I'm not going to merge this PR directly, but rather use it as inspiration for some changes I'm planning after the weekend. I've just been waiting on the JSCS release I see is now out :)

**Comment by markelog** [comment]
ok-ok

**Pull request closed without merging**